File: BibliotecaDevlights.Data/Entities/Order.cs
namespace BibliotecaDevlights.Data.Entities
{
    public class Order
    {
        public int Id { get; set; }
        public int UserId { get; set; }
        public DateTime OrderDate { get; set; }
        public decimal TotalAmount { get; set; }
        public string Status { get; set; } = string.Empty;
        public User? User { get; set; }
        public ICollection<OrderItem>? OrderItems { get; set; } 
    }
}

File: frontend/my-app/features/types/order.ts
import { OrderItems } from "./orderItems";

export interface Order{
    id:string;
    userId:string;
    orderDate:string;
    totalAmount:number;
    status:string;
    items: OrderItems[];
}

export interface OrderDto{
    userId:string;
    orderDate:string;
    totalAmount:number;
    status:string;
    items: OrderItems[];
}